config/rbac/role.yaml

```diff
@@ -27,6 +27,16 @@ rules:
   - nodes
   verbs:
   - list
+- apiGroups:
+  - ""
+  resources:
+  - pods
```

Comment:
we need to move this to a role instead of cluster role
this gives way too many privileges to the controller

Reply:
I don't think this is needed anymore since the controller is building a remote client from the kubeconfig stored on the cluster which has admin on workload clusters. I'll do some testing and remove it